src/Features/VisualBasic/Completion/CompletionProviders/SymbolCompletionProvider.vb

```diff
@@ -61,11 +62,20 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Completion.Providers
             ' don't want to trigger after a number.  All other cases after dot are ok.
             Dim root = Await document.GetVisualBasicSyntaxRootAsync(cancellationToken).ConfigureAwait(False)
             Dim token = root.FindToken(characterPosition)
-            If token.Kind = SyntaxKind.DotToken Then
-                token = token.GetPreviousToken()
+            Return IsValidTriggerToken(token)
+        End Function
+
+        Private Function IsValidTriggerToken(token As SyntaxToken) As Boolean
+            If token.Kind <> SyntaxKind.DotToken Then
+                Return False
             End If
 
-            Return token.Kind <> SyntaxKind.IntegerLiteralToken
+            Dim previousToken = token.GetPreviousToken()
+            If previousToken.Kind = SyntaxKind.IntegerLiteralToken Then
+                Return token.Parent.Kind <> SyntaxKind.SimpleMemberAccessExpression OrElse DirectCast(token.Parent, MemberAccessExpressionSyntax).Name.Kind() <> SyntaxKind.NumericLiteralExpression
```

Comment:
How can the MemberAccessExpressionSyntax's "Name" have a kind of NumericLiteralExpression? Did you mean to check the Expression (left side of the dot) instead of the Name (right side)?